pagopa/io-app
# feat(Sicilia Vola): [IASV-28] Add check income sick screen

## This PR depends on #3278 
## Short description
This PR integrate the `CheckIncomeComponent` in the `SickCheckIncomeScreen`.

## List of changes proposed in this pull request
- Insert `CheckIncomeComponent`
- Remove unused props and imports


https://user-images.githubusercontent.com/11773070/128162286-3370f2ca-aed5-4cd6-859e-4464ee046050.mp4


## How to test
Modify in the `CheckStatusRouterScreen` the following lines:
https://github.com/pagopa/io-app/blob/0150ad3c78a7b842f02d929ab88ca5b43ce0bf23/ts/features/bonus/siciliaVola/screens/voucherGeneration/CheckStatusRouterScreen.tsx#L99-L100
in:
```
isServiceAlive: remoteReady(true),
tosAccepted: remoteReady(true)
```
Starts the flow from the SiciliaVola service detail and select in:
- `CheckStatusRouterScreen` -> `Resident in Sicily`
- `SelectBeneficiaryCategoryScreen` -> `Sick`

Make sure you have the updated version of the master branch on dev-server in order to see the service details.
